pco/src/data_types/floats.rs

```diff
@@ -2,29 +2,47 @@ use std::mem;
 
 use crate::constants::Bitlen;
 use crate::data_types::{split_latents_classic, FloatLike, Latent, NumberLike};
-use crate::{float_mult_utils, ChunkConfig, FloatMultSpec, Mode};
+use crate::{float_mult_utils, float_decomp_utils, ChunkConfig, FloatMultSpec, Mode};
 
 fn choose_mode_and_split_latents<F: FloatLike>(
   nums: &[F],
   chunk_config: &ChunkConfig,
 ) -> (Mode<F::L>, Vec<Vec<F::L>>) {
-  match chunk_config.float_mult_spec {
-    FloatMultSpec::Enabled => {
-      if let Some(fm_config) = float_mult_utils::choose_config(nums) {
-        let mode = Mode::float_mult(fm_config.base);
-        let latents = float_mult_utils::split_latents(nums, fm_config.base, fm_config.inv_base);
+  if chunk_config.float_decomp_spec != FloatDecompSpec::Disabled {
```

Comment:
Let's not touch this part yet. For now it will be impossible to choose Quantize mode, but that's ok.

When we do get to this, I think the logic will look more like
```
match (fm_config, quantize_config) {
  // one provided => return that
  // both provided => error
  // either enabled => do some common stuff, then maybe choose one or return classic
  // disabled => classic
}
```

Reply:
I'll see what kind of lightweight feature flag makes sense when I go to add unit tests.

Reply:
Filed https://github.com/mwlon/pcodec/issues/194 to track implementing the full version of the choose logic.